index.html

```diff
@@ -3088,21 +3088,21 @@ <h2 id="OfflineAudioCompletionEvent">
           </dl>
         </section>
         <section>
-          <h3 id="lifetime-AudioNode" class="informative">
+          <h3 id="lifetime-AudioNode">
             Lifetime
           </h3>
           <p>
-            <i>This section is informative.</i>
+            The following behaviors provide a normative description of the
+            conditions under which an AudioNode is alive, meaning that it MUST
+            be retained in the graph by an implementation. Where these
+            conditions do not apply, AudioNodes MAY be released by an
+            implementation. Additionally, implementations may apply other
+            methods to avoid unnecessary resource usage and unbounded memory
+            growth of unused/finished nodes.
```

Comment:
Dunno. Just leave out the last sentence?  Of the couple of languages that I know that  have GC, GC isn't actually required as part of the language. Maybe javascript is one such language so we don't have to say anything about GCing "unused/finished" node?